docs/products/known_issues_and_subtleties.rst

```diff
@@ -16,6 +16,45 @@ We are currently integrating a major change to our deblender that we hope will m
 
 It is likely that our direct image processing in crowded fields will still lag dedicated crowded-field photometry codes; Rubin's focus for these fields has always been image subtraction (where we do expect to compete with the state of the art), with the :ref:`catalogs-object` catalog a best-effort addition.
 
+.. _products_wcs_known_issues:
+
+WCS FITS approximations and misleading interfaces
+=================================================
+
+Rubin's single-visit World Coordinate System (WCS) objects are not in general exactly representable via the FITS WCS standard, and hence the FITS WCS in the headers of the :ref:`visit image <images-visit-image>` and :ref:`difference image <images-difference-image>` data products (they are the same) are approximations that are only expected to be good enough for visualization and object-finding, not precision astrometry.
```

Comment:
how about making each section for visit/difference images, coadds, and raw images to improve clarity?